src/it/resources/org/checkstyle/suppressionxpathfilter/throwscount/SuppressionXpathRegressionThrowsCount2.java

```diff
@@ -0,0 +1,22 @@
+package org.checkstyle.suppressionxpathfilter.throwscount;
+
+public class SuppressionXpathRegressionThrowsCount2 {
+    public void myFunction() throws IllegalStateException, // warn, max allowed is 2
+            ArrayIndexOutOfBoundsException,
```

Comment:
this Input is identical to previous.
No much value in it, please do violation on methods of interface at least.

Reply:
Should I add this method under an interface?
like this: 
```
interface myInterface {
        public void myFunction() throws IllegalStateException, // warn, max allowed is 2
                ArrayIndexOutOfBoundsException,
                NullPointerException;
    }
```
as this method doesn't have a body too.
Xpath query is also different now from the previous method:
```
"/COMPILATION_UNIT"
          + "/CLASS_DEF[./IDENT[@text='SuppressionXpathRegressionThrowsCount2']]"
          + "/OBJBLOCK/INTERFACE_DEF[./IDENT[@text='myInterface']]"
          + "/OBJBLOCK/METHOD_DEF[./IDENT[@text='myFunction']]"
          + "/LITERAL_THROWS[./IDENT[@text='IllegalStateException']]"
```

Reply:
Yes please